### ISSUE TITLE
[BUG] Incorrect version in CMakeLists for release 2.1.18

### ISSUE THREAD

**Issue by dtrodrigues** [OpenImageIO/oiio#2651]
**Describe the bug**
The [version string](https://github.com/OpenImageIO/oiio/blob/3178f23db037c4864cfd21737055f7d5f1ff5ee1/CMakeLists.txt) in [Release 2.1.18](https://github.com/OpenImageIO/oiio/releases/tag/Release-2.1.18.0) has the previous version (2.1.17).

**To Reproduce**
Untar and build the 2.1.18 release: https://github.com/OpenImageIO/oiio/archive/Release-2.1.18.0.tar.gz 

```
>>> import OpenImageIO
>>> print(OpenImageIO.VERSION_STRING)
2.1.17
```

**Expected behavior**
The version number to match the release version.

**Evidence**
Do you have error messages? (please quote exacty) Screenshots? Example
command lines or scripts that reliably reproduce the error? If it only
happens with certain image files, can you attach the smallest image you
can make that reproduces the problem?
https://github.com/Homebrew/homebrew-core/pull/58975/checks?check_run_id=934574220

**Platform information:**
 - OIIO branch/version: 2.1.18
 - OS: macOS

**Comment by lgritz** [comment]
Oh dear, you are correct. Oops. Will fix right away and tag Release-2.1.18.1.

**Comment by lgritz** [comment]
Fixed and pushed out a new Release-2.1.18.1
